playbooks/appsemblerPlaybooks/dogwood_multiserver_deploy_cloud_sql.yml

```diff
@@ -28,7 +28,7 @@
     COMMON_ENABLE_DATADOG: False
   roles:
     - { role: swapfile, SWAPFILE_SIZE: "4GB" }
-    - mysql_init
+    - { role: mysql_init, mysql_cert_owner: "edxapp" }
```

Comment:
@melvinsoft Good point. I was thinking that migrations are only run by `edxapp`, but I see that [xqueue also runs migrations](https://github.com/appsembler/configuration/blob/3e7c4e7c46cbb1690c83fb416bad4985f561c838/playbooks/roles/xqueue/tasks/deploy.yml#L90-L99).

A better approach might be to create an owner/group for the certs, then add users to the group.